helper/schema/schema_test.go

```diff
@@ -5329,6 +5329,7 @@ func TestSchemaMap_Validate(t *testing.T) {
 				},
 			},
 		},
+		*/
```

Comment:
Out of curiosity, does this test pass if you add `Err: True`? I'm wondering if this is one of those cases where HCL2 (or some other update) is helpfully catching the error sooner than before (but not curious enough to test locally ;) )